physical_infra_provider_configuration/configuration/topics/installing_and_configuring_lxca.adoc

```diff
@@ -1 +1,47 @@
-=== Install and Configure LXCA
\ No newline at end of file
+=== Install and Configure Lenovo XClarity Administrator
+
+The Lenovo XClarity Administrator Provider leverages one or more Lenovo XClarity Administrator appliances to centralize resource-management solution and enhance the availability of Lenovo® server systems and solutions. 
+
+Lenovo XClarity Administrator is available stand-alone or as a bundled offering that is known as Lenovo XClarity Pro. Lenovo XClarity Pro is composed of the base Administrator product plus two Lenovo XClarity Integrator modules that provide integration into Microsoft Systems Center or VMware vCenter. Together, these tools provide discovery, monitoring, configuration, and management functions to reduce the cost and complexity of routine system administration for supported devices.
+
+For more information downloading the Lenovo XClarity Appliance or obtaining a license for Lenovo XClarity Pro reference the following link:
+
+* Lenovo XClarity Administrator 
+(https://www3.lenovo.com/us/en/data-center/software/systems-management/xclarity)
+
+==== Installation
+
+Lenovo XClarity Administrator is delivered as a virtual appliance that supports industry leading virtualization plaforms (ESXi, Hyper-V, Nutanix). Follow the instructions for importing a virtual machine as provided by your platform.  
+
+
+==== Configuration
+
+When you access Lenovo XClarity Administrator for the first time, there are several steps that you must complete to initially set up Lenovo XClarity Administrator. Complete the following steps to initially configure XClarity Administrator:
+
+
+===== Procedure
+
+* Open a web browser.
```

Comment:
Maybe the steps of the procedure should be numbered. What do you think?

Reply:
That is fair, I will update the bullets.